cpp/src/toppra/toppra.hpp

```diff
@@ -54,6 +54,12 @@ namespace toppra {
   /// Vector of Bound
   typedef std::vector<Bound, Eigen::aligned_allocator<Bound> > Bounds;
 
+  /// Boundary condition
+  typedef struct {
```

Comment:
Since this boundary condition is required only for PieceWisePolynomial, I suggest putting it in the `piecewise_poly_path.hpp` header file.

Reply:
Noted. Changed in https://github.com/hungpham2511/toppra/pull/161/commits/9023de3c13e92777776f6dd9b0b05fb78f2fbe84. Thanks.